chat-formatter.js

```diff
@@ -87,7 +87,7 @@ class TextFormatter {
 		this.stack = /** @type {FormatSpan[]} */ ([]);
 		this.isTrusted = isTrusted;
 		/** offset of str that's been parsed so far */
-		// @ts-ignore TypeScript bug: can't infer type
+		/** @type {number} */
```

Comment:
Is the TypeScript bug still present? We should report it probably...

Reply:
Yeah, it still doesn't infer that. For now, compromise and leave the type declaration and also a comment saying it's a bug?

Reply:
sure